src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Release History
 
 * Add support for importing/exporting feature flags
 
+**ARM**
+
+* Fix Recovery Services Vault tags cannot be updated
+
 2.0.78
```

Comment:
Why two 2.0.78 here? Please help fixed it.

Reply:
Done